Library/Formula/megatools.rb

```diff
@@ -27,7 +28,7 @@ def install
   test do
     # Downloads a publicly hosted file and verifies its contents.
     system "#{bin}/megadl",
-      "https://mega.co.nz/#!3Q5CnDCb!PivMgZPyf6aFnCxJhgFLX1h9uUTy9ehoGrEcAkGZSaI",
+      "https://mega.nz/#!3Q5CnDCb!PivMgZPyf6aFnCxJhgFLX1h9uUTy9ehoGrEcAkGZSaI",
```

Comment:
Is this chunk necessary? It looks to be redirected automatically. Feel free to send this chunk upstream to `Homebrew/homebrew` and remove it here.